syntax = "proto3";

import "google/protobuf/empty.proto";

service ClockService {
  rpc fetch_clock(google.protobuf.Empty) returns (FetchClockResponse) {}
  rpc adjust_clock(AdjustClockRequest) returns (google.protobuf.Empty) {}
}

message AdjustClockRequest {
    double offset = 1;
}

message FetchClockResponse {
    double timestamp = 1;
}
